pom.docker.xml

```diff
@@ -1058,7 +1058,7 @@
         </repository>
     </repositories>
     <properties>
-        <swagger-codegen-generators-version>1.0.28-SNAPSHOT</swagger-codegen-generators-version>
+        <swagger-codegen-generators-version>3.0.29-SNAPSHOT</swagger-codegen-generators-version>
```

Comment:
A `3.0.29-SNAPSHOT` version for `swagger-codegen-generators` repo does not exist. You can check the releases versions [here](https://mvnrepository.com/artifact/io.swagger.codegen.v3/swagger-codegen-generators)